setupKAppNavLib.sh

```diff
@@ -43,10 +58,20 @@ handleParams() {
 
 build() {
     echo "########## Build started on $(date)  ##########"
-    ./build.sh
-    if [ $? -ne 0 ]; then
-        echo "########## Error: build failed ##########"
-        exit 1
+    if [ x$reposArg = 'x' ] || [ x$reposArg = "" ]; then
```

Comment:
Oh, I see now, it's individual calls to build.sh when `repos` is set.  We could still eliminate the difference if `build.sh` were updated to take a list instead of a single repo.